google/cloud/storage/internal/grpc/configure_client_context_test.cc

```diff
@@ -192,6 +192,44 @@ TEST_F(GrpcConfigureClientContext, ApplyRoutingHeadersUploadChunkNoMatch) {
   EXPECT_THAT(metadata, Not(Contains(Pair("x-goog-request-params", _))));
 }
 
+TEST_F(GrpcConfigureClientContext, ApplyRoutingHeadersUploadId) {
+  using ::testing::Eq;
+  using ::testing::Matcher;
+
+  struct TestCase {
+    std::string upload_id;
+    ::testing::Matcher<std::string> matcher;
```

Comment:
nit: s/::testing//

Reply:
Although, I would make these things `std::string`s and let GoogleMock do its magic. 🤷

Reply:
Fixed.